csuntechlab/checkpoint
# Check 71 - Create organization creation form on admin site

# Description
- Installed Vuetify
- Added Nav bar
- Added Signup page
- Update router

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Load up your environment `docker-compose up -d`
- Run yarn
- Run `yarn run dev`
- Visit [http://localhost:8080](localhost:8080) and verify the sign up page matches the paper mockup

# Checklist:
